Saturation/Splitter.cpp

```diff
@@ -703,9 +703,16 @@ void Splitter::init(SaturationAlgorithm* sa)
 #endif
 
   if (opts.splittingAvatimer() < 1.0) {
-    _stopSplittingAt = opts.splittingAvatimer() * opts.timeLimitInDeciseconds() * 100;
+    _stopSplittingAtTime = opts.splittingAvatimer() * opts.timeLimitInDeciseconds() * 100;
+    _stopSplittingAtInst = 0;
+#ifdef __linux__
```

Comment:
If this kind of thing is going to proliferate, maybe we should introduce a new preprocessor define like `INSTRUCTION_COUNT_AVAILABLE`?

Reply:
A good point. I was just too lazy for this one. But will do it with the next!